types/project.go

```diff
@@ -102,10 +104,19 @@ func (p *Project) ConfigNames() []string {
 
 // GetServices retrieve services by names, or return all services if no name specified
 func (p *Project) GetServices(names ...string) (Services, error) {
+	services, servicesNotFound := p.getServicesByNames(names...)
+	if len(servicesNotFound) > 0 {
+		return services, fmt.Errorf("no such service: %s", servicesNotFound[0])
```

Comment:
nit: we should return all not found services at once, otherwise if people have multiple services missing they'll have to go

```shell
$ compose up
[no such service: foo]

<fixes config>

$ compose up
[no such service: bar]

<yells at computer and fixes things again>
```

Reply:
As this method is public, I didn't want to introduce a breaking change by adding the support `depends_on.required` attribute, so this implementation keep the same behaviour as the current one.
I agree we should return all the missing service at once, but I prefer to do that in a dedicated PR if this is fine for you @laurazard?